src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -1215,6 +1218,20 @@ function setTooltipTriggerOnFeedbackPathMenuOptions() {
     });
 }
 
+/**
+ * Enables options of different question types
+ * to be reordered using a drag and drog mechanism
+ */
+function makeQuestionOptionsReorderable() {
+    const numQuestions = $('.questionTable').length;
+    for (let i = 1; i <= numQuestions; i += 1) {
```

Comment:
can use `i++`?

Reply:
umm we seem to be using `i += 1` everywhere so wanted to be consistent ? Its more explicit as well